queries.sql

```diff
@@ -8,4 +8,40 @@ select date_of_birth from animals where name = 'Agumon' OR name = 'Pikachu';
 select name, escape_attempts from animals where weight_kg > 10.5;
 select * from animals where neutered = true;
 select * from animals where name != 'Gabumon';
-select * from animals where weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+select * from animals where weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT name, species FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' where name like '%mon';
+UPDATE animals SET species = 'pokemon' where species IS NULL;
```

Comment:
- [ ] After this transaction, kindly commit the transaction and verify that change was made and persists after commit. For example:
```
SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```
- [ ] Also, kindly take a screenshot before and after the transactions according to the requirements.

Reply:
DONE

Reply:
This issue is well resolved! ✔️